observation_portal/observations/serializers.py

```diff
@@ -501,10 +507,15 @@ def validate(self, data):
 
         if self.context.get('request').method == 'PATCH':
             # For a partial update, only validate that the 'end' field is set, and that it is > now
-            if 'end' not in data:
-                raise serializers.ValidationError(_('Observation update must include `end` field'))
-            if data['end'] <= timezone.now():
+            if 'end' not in data and 'state' not in data:
+                raise serializers.ValidationError(_('Observation update must include `end` or `state` field'))
+            if 'end' in data and data['end'] <= timezone.now():
                 raise serializers.ValidationError(_('Updated end time must be in the future'))
+            if 'state' in data:
+                if data['state'] != 'BAD_DATA':
+                    raise serializers.ValidationError(_('Users can only update Observation state to BAD_DATA'))
```

Comment:
Might make sense to make this passive.
"Observation state can only be updated to BAD_DATA"